### PR TITLE
Register warden tete110

### PR DESCRIPTION
**Pull request by code423n4**
Auto-generated PR to register the new warden tete110

### CODE REVIEW

**Comment by SamusElderg** [comment]
Have contacted user to change their nickname to match application

File: _data/handles/tete110.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "handle": "tete110",
```
Comment (GabrielKomado) [inline _data/handles/tete110.json:2]: hi

Comment (GabrielKomado) [reply _data/handles/tete110.json:2]: this is me

File: _data/handles/tete110.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+  "link": "",
+  "moralisId": "7L46aNCnSG1x7C40w2Zc2Eb9"
+}
```
Comment (GabrielKomado) [inline _data/handles/tete110.json:5]: hi

**Review by GabrielKomado — APPROVED** [review]

**Review by GabrielKomado — APPROVED** [review]
hi

**Review by kartoonjoy — APPROVED** [review]

**Pull request merged**